src/sentry/static/sentry/app/utils/getDynamicText.tsx

```diff
@@ -2,6 +2,12 @@
 
 // Return a specified "fixed" string when we are in a testing environment
 // (more specifically in a PERCY env (e.g. CI))
-export default function getDynamicText({value, fixed}) {
+export default function getDynamicText({
+  value,
+  fixed,
+}: {
+  value: string;
```

Comment:
Isn't value sometimes a node/element? We use this function for replacing graphs & video elements too.

Reply:
Ah good catch. Updated